src/graph_world/nodeclassification/generator_wrappers.py

```diff
@@ -28,6 +28,7 @@ class SbmGeneratorWrapper(GeneratorConfigSampler):
   def __init__(self, param_sampler_specs, marginal=False,
                normalize_features=True, use_generated_lfr_communities=False, lfr_params=None):
     super(SbmGeneratorWrapper, self).__init__(param_sampler_specs)
+    self._AddParamSamplerSpecs(lfr_params, use_generated_lfr_communities) 
```

Comment:
Please change the location and design of this fn:

(1) Location: since something like this is only needed on non-LFR generator wrappers, define it as a private member function of this class. 

(2) Design: Currently this function overwrites *all* the SBM parameters when `use_generated_lfr_communities=True`, which is not what we want. Instead:
  * if `use_generated_lfr_communities=True`, add the 4 LFR community param `{spec.name: spec}` dictionary items to `self._param_sampler_specs`
  * if `use_generated_lfr_communities=False`, do nothing
  
Also, please see current L48, where the LFR sampler fns are added. It is possible to do the above and add the sampler fns all in one function. This might be more readable, as all the extra LFR parameter names will be in one place.